tests/integration/targets/aws_s3_bucket_info/tasks/main.yml

```
@@ -0,0 +1,92 @@
+---
+- name: Test community.aws.aws_s3_bucket_info
+  module_defaults:
+    group/aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+  block:
+    - name: Create simple s3_buckets
+      s3_bucket:
+        name: "{{ item }}"
+        state: present
+        tags:
+          integration: "verified"
+          facts: "defined"
```

We often mangle tags accidentally (either when getting or setting them) I'd like to see 4 examples:
```suggestion
          "lowercase spaced": 'hello cruel world'
          "Title Case": 'Hello Cruel World'
          CamelCase: 'SimpleCamelCase'
          snake_case: 'simple_snake_case'
```

These should come back out of _info unchanged...

---

https://github.com/zeten30/community.aws/commit/e5db616dc5ea9cb5829d5ae56bd90ff3d13f0df4